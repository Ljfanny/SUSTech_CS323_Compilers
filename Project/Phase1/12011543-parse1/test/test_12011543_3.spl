int test_12011543_3()
{
  int a, bb
  if (a > 4)
  	a = c | 4;
  	b = a & 5;
    return bb;
}