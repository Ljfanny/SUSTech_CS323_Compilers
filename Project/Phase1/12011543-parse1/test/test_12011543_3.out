Error type B at Line 3: Missing semicolon ';'
Error type A at Line 5: unknown lexeme |
Error type A at Line 6: unknown lexeme &
